site/_quarto.yml

```diff
@@ -34,58 +34,56 @@ website:
     title: false
     pinned: true
 
-    left:
-      - text: "About"
-        file: about/overview.qmd
-      - text: "Get Started"
-        file: get-started/get-started.qmd
-      - text: "Guides"
-        file: guide/guides.qmd
-      - text: "{{< fa cube >}} {{< var validmind.developer >}}"
-        menu:
-        - text: "{{< fa laptop-code >}} {{< var validmind.developer >}}"
-          file: developer/validmind-library.qmd
-        - text: "{{< fa cubes >}} Supported Models"
-          file: developer/supported-models.qmd
-        - text: "---"
-        - text:  "{{< fa rocket >}} QUICKSTART"
-        - text: "{{< fa file-pen >}} For Model Documentation"
-          file: notebooks/quickstart/quickstart_model_documentation.ipynb
-        - text: "{{< fa clipboard-check >}} For Model Validation"
-          file: notebooks/quickstart/quickstart_model_validation.ipynb
-        - text: "---"
-        - text:  "{{< fa vial >}} TESTING"
-        - text: "{{< fa flask-vial >}} Run Tests & Test Suites"
-          file: developer/model-testing/testing-overview.qmd
-        - text: "{{< fa microscope >}} Test Descriptions"
-          file: developer/model-testing/test-descriptions.qmd
-        - text: "{{< fa toolbox >}} Test Sandbox (BETA)"
-          file: developer/model-testing/test-sandbox.qmd
-        - text: "---"
-        - text: "{{< fa code >}} CODE SAMPLES"
-        - text: "{{< fa book-open-reader >}} All Code Samples · `LLM` · `NLP` · `Time Series` · `Etc.`"
-          file: developer/samples-jupyter-notebooks.qmd
-        - text: "{{< fa download >}} Download Code Samples · `notebooks.zip`"
-          file: notebooks.zip
-        - text: "{{< fa hand-point-right >}} Try it on JupyterHub {{< fa hand-point-left >}}"
-          file: https://jupyterhub.validmind.ai/
-        - text: "---"
-        - text: "{{< fa book >}} REFERENCE"
-        - text: "{{< var validmind.api >}}"
-          file: validmind/validmind.qmd
-      - text: "Support"
-        file: support/support.qmd
-      - text: "Training"
-        file: training/training.qmd
-      - text: "Releases"
-        file: releases/all-releases.qmd
-      # - text: "validmind.com {{< fa external-link >}}"
-      #   file: https://validmind.com/
-      #   target: _blank
+    # left:
```

Comment:
Remove commented out stuff from _quarto.yml if you don't need it anymore, the resulting file should be clean of cruft. (Also see my other comment about fixing the børked navigation.)